Gun.cpp

```diff
@@ -0,0 +1,30 @@
+// Gun.cpp
+// Sheng Fang, Rabiya Majeed, & Zulaikha Zakiullah
+// This instantiates the gun class, the gun [|87.
+
+#include "Gun.h"
+
+Gun::Gun() {
+	Gun::ammo = 0;
+	Gun::durability = 0;
+	Gun::shooting_speed = 0;
+	Gun::accuracy = 0;
+	Gun::durability = 0;
+	Gun::recoil = 0;
+}
+
+Gun::~Gun() {}
+
+int Gun::get_ammo() { return Gun::ammo; }
+int Gun::get_durability() { return Gun::durability; }
+int Gun::get_shooting_speed() { return Gun::shooting_speed; }
+double Gun::get_accuracy() { return Gun::accuracy; }
+int Gun::get_durability() { return Gun::durability; }
+double Gun::get_recoil() { return Gun::recoil; }
+
+void Gun::set_ammo(int new_ammo) { Gun::ammo = ammo; }

```

Comment:
You have the parameter as "new_ammo" but you're saying Gun::ammo = ammo, which just resets it to itself. I think you should change the parameter to just "ammo" in the .cpp and .h file (same for the other set functions)